### PR TITLE
Add instructions for remote ray

### PR DESCRIPTION
**Pull request by ahuang11**
## Summary

Added instructions for fixing: `PermissionError: [Errno 13] Permission denied: '/Users'`

## Relevant Issue(s)

https://github.com/PrefectHQ/prefect-ray/issues/26

## Checklist
- [ ] Summarized PR's changes in [CHANGELOG.md](https://github.com/PrefectHQ/prefect-ray/blob/main/CHANGELOG.md)

### CODE REVIEW

**Comment by ahuang11** [comment]
Tests fixed in https://github.com/PrefectHQ/prefect-ray/pull/48

File: README.md [hunk 1]
```diff
@@ -119,6 +119,22 @@ Note that Ray Client uses the [ray://](https://docs.ray.io/en/master/cluster/ray
 
     See the [Ray installation documentation](https://docs.ray.io/en/latest/ray-overview/installation.html) for further compatibility information.
 
+## Running tasks on a Ray remote cluster
+
```
Comment (desertaxle) [inline README.md:123]:
```suggestion

When using the `RayTaskRunner` with a remote Ray cluster, you may run into issues that are not seen when using a local Ray instance. To resolve these issues, we recommend taking the following steps when working with a remote Ray cluster:

```

File: README.md [hunk 2]
```diff
@@ -119,6 +119,22 @@ Note that Ray Client uses the [ray://](https://docs.ray.io/en/master/cluster/ray
...
+1. Set `PREFECT_LOCAL_STORAGE_PATH` to a path accessible on both the remote cluster and the machine executing the flow:
```
Comment (desertaxle) [inline README.md:124]:
```suggestion
1. If your task fail with permission or file not found errors, set `PREFECT_LOCAL_STORAGE_PATH` in your Prefect settings to a path accessible on both the remote cluster and the machine executing the flow:
```

File: README.md [hunk 3]
```diff
@@ -119,6 +119,22 @@ Note that Ray Client uses the [ray://](https://docs.ray.io/en/master/cluster/ray
...
+```bash
+prefect config set PREFECT_LOCAL_STORAGE_PATH='/tmp/prefect/storage'
+```
+
+2. Ensure `prefect` is installed on the remote cluster, with:
```
Comment (desertaxle) [inline README.md:129]:
```suggestion
2. If you get an error stating that the module 'prefet' cannot be found, ensure `prefect` is installed on the remote cluster, with:
```

File: README.md [hunk 4]
```diff
@@ -119,6 +119,22 @@ Note that Ray Client uses the [ray://](https://docs.ray.io/en/master/cluster/ray
...
+```bash
+pip install prefect
+```
+
+3. Pass `address='ray://<head_node_ip_address>:10001'`:
```
Comment (desertaxle) [inline README.md:134]:
```suggestion
3. If you are seeing timeout or other connection errors, double check the address provided to the `RayTaskRunner`. The address should look similar to: `address='ray://<head_node_ip_address>:10001'`:
```

**Review by desertaxle — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -119,6 +119,24 @@ Note that Ray Client uses the [ray://](https://docs.ray.io/en/master/cluster/ray
 
     See the [Ray installation documentation](https://docs.ray.io/en/latest/ray-overview/installation.html) for further compatibility information.
 
+## Running tasks on a Ray remote cluster
+
+When using the `RayTaskRunner` with a remote Ray cluster, you may run into issues that are not seen when using a local Ray instance. To resolve these issues, we recommend taking the following steps when working with a remote Ray cluster:
+
+1. If your task fail with permission or file not found errors, set `PREFECT_LOCAL_STORAGE_PATH` in your Prefect settings to a path accessible on both the remote cluster and the machine executing the flow:
```
Comment (ahuang11) [inline README.md:126]:
```suggestion
1. If your task fails with permission or file not found errors, set `PREFECT_LOCAL_STORAGE_PATH` in your Prefect settings to a path accessible on both the remote cluster and the machine executing the flow:
```

File: README.md [hunk 6]
```diff
@@ -119,6 +119,24 @@ Note that Ray Client uses the [ray://](https://docs.ray.io/en/master/cluster/ray
...
+## Running tasks on a Ray remote cluster
+
+When using the `RayTaskRunner` with a remote Ray cluster, you may run into issues that are not seen when using a local Ray instance. To resolve these issues, we recommend taking the following steps when working with a remote Ray cluster:
+
+1. If your task fails with permission or file not found errors, set `PREFECT_LOCAL_STORAGE_PATH` in your Prefect settings to a path accessible on both the remote cluster and the machine executing the flow:
+```bash
+prefect config set PREFECT_LOCAL_STORAGE_PATH='/tmp/prefect/storage'
+```
+
+2. If you get an error stating that the module 'prefet' cannot be found, ensure `prefect` is installed on the remote cluster, with:
```
Comment (ahuang11) [inline README.md:131]:
```suggestion
2. If you get an error stating that the module 'prefect' cannot be found, ensure `prefect` is installed on the remote cluster, with:
```

**Review by tekumara — APPROVED** [review]

**Review by desertaxle — APPROVED** [review]

**Pull request merged**